### DISCUSSION TITLE
undefined reference to `drmModeCloseFB'

### DISCUSSION THREAD

**Discussion by erentasken** [hyprwm/Hyprland#4960] [Q&A]
/usr/bin/ld: /home/eren/Downloads/Hyprland/subprojects/wlroots/build/libwlroots.so: undefined reference to `drmModeCloseFB'
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
make[1]: *** [Makefile:15: release] Error 1
make[1]: Leaving directory '/home/eren/Downloads/Hyprland'
make: *** [Makefile:31: all] Error 2

**Comment by jlecount** [comment]
I'm also seeing this.  @erentasken did you ever get past it?
